src/go/rpk/pkg/cli/cmd/wasm/common.go

```diff
@@ -105,3 +113,21 @@ func CreateRemoveMsg(name string) sarama.ProducerMessage {
 		Headers: headers,
 	}
 }
+
+func AddTypeFlag(command *cobra.Command, coprocType *string) {
+	command.Flags().StringVar(
+		coprocType,
+		"type",
+		"async",
+		"Coproc type engine (async, data-policy)",
+	)
+}
+
+func CheckCoprocType(coprocType string) error {
+	switch coprocType {
+	case "async", "data-policy":
+		return nil
+	default:
+		return fmt.Errorf("Unexpected coproc type: '%s'", coprocType)
```

Comment:
Are we using the term "coproc" for user-facing text? I thought we had deprecated it.

Reply:
Hm, yep

I think `wasm type`will be better